torch/_dynamo/symbolic_convert.py

```diff
@@ -2131,6 +2131,7 @@ def _throw_if_in_functorch(self):
         forbidden_keys = (
             torch._C._functorch.TransformType.Vmap,
             torch._C._functorch.TransformType.Grad,
+            torch._C._functorch.TransformType.Jvp,
```

Comment:
@guilhermeleobas did you get this PR to work without adding logic to fake-ify the tangent of a GradTrackingTensor?

Reply:
Yes!